includes/rest-api/class-sensei-rest-api-data-port-base.php

```diff
@@ -0,0 +1,199 @@
+<?php
+/**
+ * Import/Export REST API Base Controller.
+ *
+ * @package Sensei\DataPort
+ * @since   3.1.0
+ */
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit; // Exit if accessed directly.
+}
+
+/**
+ * Base class for Sensei's Import/Export REST API controllers.
+ *
+ * @package Sensei
+ * @author  Automattic
+ * @since   3.1.0
+ */
+abstract class Sensei_REST_API_Data_Port_Base extends \WP_REST_Controller {
```

Comment:
I think that `Sensei_REST_API_Data_Port_Controller` describes better what this class is.

Reply:
I like it. 50c3878edc88d9f5cb138023ffc12c1b3f977b96